src/workflows/airqo_etl_utils/message_broker_utils.py

```
@@ -1,46 +1,78 @@
 import json
 import logging
+import sys
 
 import numpy as np
 import pandas as pd
-from kafka import KafkaProducer
+from confluent_kafka import Producer, Consumer, TopicPartition
+import time
```

Ruff

`time` imported but unused

Remove unused import: `time`

(F401)